src/app/shared/services/auth.service.ts

```diff
@@ -93,13 +93,12 @@ export class AuthService {
   }
 
   logout(): void {
-    localStorage.removeItem('token');
-    localStorage.removeItem('userId');
-    localStorage.removeItem('stayLoggedIn');
+    localStorage.clear();
     // Update login status to false
     this.loggedIn.next(false);
     // Navigate to the home page
     this.router.navigate(['/']);
+    window.location.reload();
     // TO DO : implémenter méthode logout côté back
     // this.http.post(`${API_URL}/logout`, {}).subscribe({
```

Comment:
Je pense qu'on doit enlever ce TODO, juste pour ne plus nous embrouiller sur ce sujet:
Il n'y a pas besoin d'une méthode logout dans le back end. 
Le logout se fait explicitement du côté front, par 'localStorage.clear();'.
Dans le back, il n y a pas d'état 'logged in/logged out', donc en faisant simplement une requête sans JWT (ou contenant un JWT non valide), le back refusera implicitement l'authentification, et ne permettra pas l'accès aux ressources demandées

Reply:
Ok si on en est sûr, je supprime :)